tests/model/test_model.py

```diff
@@ -2079,20 +2098,73 @@ def test_toggle_stream_pinned_status(
                     "id": 0,
                 },
                 True,
+                False,
             ),
         ],
-        ids=["not_in_pm_narrow", "not_in_pm_narrow_with_sender", "start", "stop"],
+        ids=[
+            "not_in_pm_narrow",
+            "not_in_pm_narrow_with_sender",
+            "start",
+            "start when animating",
+            "stop",
+        ],
     )
-    def test__handle_typing_event(self, mocker, model, narrow, event, called):
+    def test__handle_typing_event(
+        self,
+        mocker,
+        model,
+        narrow,
+        event,
+        is_animation_in_progress,
+        is_recipient_actually_typing,
```

Comment:
Let's use an `expected_` as prefix for params that are expected from the test result, so `is_recipient_actually_typing` -> `expected_is_recipient_typing`.

Reply:
:+1: